docs/site/themes/template/layouts/partials/use-cases.html

```diff
@@ -23,7 +23,7 @@ <h3>Evaluate</h3>
                 </div>
             </div>
         </a>
-        <a href="/docs/">
+        <a href="/resources">
```

Comment:
Why are we changing this from docs to resources?

Reply:
we currently link to docs from 2 of the three use case boxes here, and wanted to give the visitors more variety in their exploration path.  Linking this 'use' use case to the resources tab is a way to do that. More importantly, we are starting to build a good content library that will (hopefully soon!) show up in our resources page, where it will be organized by persona and use case - so in the mid to longer term that page should be the ideal place for people to go to land if they'd like help re: how to use TCE to solve their problems.

Reply:
I think "Learn" fits better for /resources, and "Use" can still go to /docs.